Week1/homework/queries.js

```diff
@@ -0,0 +1,26 @@
+const mySQL = require('mysql');
+const connection = mySQL.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'myWorld',
+});
+
+const queries = [
+  `select name from country where population > 8000000;`,
+  `select name from country where name like '%land%';`,
+  `select name from city where population > 50000 or population < 1000000;`,

```

Comment:
Be careful here you get population > 5000 OR population < 1000000. What if my population is 5000000 ? 
It will be selected it because 5000000 > 5000 but you don't want to select this row. Can you fix this condition?

Reply:
So true, im fixing it